app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <color name="colorPrimary">#3f51b5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
-</resources>
+<resources></resources>
```

Comment:
If you have an empty resources tag, you could remove it or you could change it to <resources />